### PR TITLE
Miscellaneous CI improvements

### PR DESCRIPTION
**Pull request by natanlao**
https://github.com/HumanCellAtlas/data-consumer-vignettes/pull/39#discussion_r311801375

### CODE REVIEW

File: test/test_output.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# test/test_output.sh 
+test $(grep -oc '"outputs": ' "$1") -eq $(grep -oc '"cell_type": "code"' "$1")
+if [ $? -gt 0 ]; then
+    echo "FAIL: Was output committed with this notebook? $1"
+fi
+exit 0
```
Comment (amarjandu) [inline test/test_output.sh]: nit, newline

File: test/test_notebook.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# test/test_output.sh path_to_notebook
+# See ReviewNB/treon#12 
+cd $(dirname $1)
+treon
```
Comment (amarjandu) [inline test/test_notebook.sh]: nit, newline

File: .travis.yml [hunk 3]
```diff
@@ -2,19 +2,24 @@ language: python
 dist: xenial  # for python3.7+ testing
 
 python:
-- 3.5
 - 3.6
```
Comment (amarjandu) [inline .travis.yml]: did you mean to drop the whole v 3.5? or just the extra dash?

Comment (natanlao) [reply .travis.yml]: Drop all of v3.5, `loompy` doesn't support it so testing against it seems to be more trouble than it's worth.

**Pull request merged**